README.md

```diff
@@ -59,8 +59,78 @@ rdataretriever::download('portal', './data/')
 portal = rdataretriever::fetch('portal')
 names(portal)
 head(portal$species)
+
 ```
 
+New Spatial data Installation
+-----------------------------
+
+**Set-up and Requirements(Tools)**
+
+**Tools**
+
+-  psql, raster2pgsql, shp2pgsql, dgal,
```

Comment:
Should `dgal` be `gdal`?

I don't see instructions below on installing PostGIS. Does that need to happen before enabling the extensions. Does it automatically install `raster2pgsql`, `shp2pgsql`, and `gdal`? If so then I might just mentioned PostgreSQL and PostGIS here. If not then we probably need to add instructions for installing those dependencies.